run/interpolators.py

```diff
@@ -112,3 +112,32 @@ def rotate_velocities(u, v, angle):
     """
 
     return (u * np.cos(angle) + v * np.sin(angle), -u * np.sin(angle) + v * np.cos(angle))
+
+def rotate_pole_to_greenland(lat, lon):
+    """
+    Rotates the mesh such that the singularities are in Greenland / Antarctica at 75N / 40W and 75S / 140E
+    This is a copy of ParametricMesh::RotatePoleToGreenland in ParametricMesh.cpp. It's here for testing purposes only.
+
+    :param lat: Latitudes of the mesh
+    :param lon: Longitudes of the mesh
+    :return: A tuple of the rotated latitudes and longitudes.
+    """
+
+    latr = np.deg2rad(lat)
+    lonr = np.deg2rad(lon)
+
+    x = np.cos(latr) * np.cos(lonr)
+    y = np.cos(latr) * np.sin(lonr)
+    z = np.sin(latr)
+
+    aw = 40.0 * np.pi / 180.0
```

Comment:
Could use 
```suggestion
    aw = np.deg2rad(40.0)
```
for consistency.